src/assets/css/base.css

```diff
@@ -6,7 +6,7 @@
   --text-primary: theme('colors.white');
   --text-secondary: theme('colors.black');
 
-  --flamingo: theme('colors.flamingo.500');
```

Comment:
Not a blocker, but instead of using a color name we could use `brand`. That way we can change the color in the future if needed. Maybe we can do a simple search and replace for this change.